web/src/popover_menus.js

```diff
@@ -441,4 +450,51 @@ export function initialize() {
             message_actions_popover_keyboard_toggle = false;
         },
     });
+
+    // Starred messages popover
+    tippy_no_propagation(".starred-messages-sidebar-menu-icon", {
+        placement: "right",
+        maxWidth: "335px",
```

Comment:
How did you pick 335px for this?

On placement, have you tested how this ends up looking in mobile UI sizes? I can imagine switching libraries causing that to improve or regress, and we might need some extra configuration to ensure the behavior is reasonable.

(From a UI standpoint, one possible option in smaller screens would be for left sidebar popovers to just cover the center of the screen if there isn't space to place them to the right)

Reply:
> How did you pick 335px for this?

This is the max size of current popover on the wide screen.

> On placement, have you tested how this ends up looking in mobile UI sizes?

In this iteration of PR, my focus was primarily on implementing the main functionality and I didn't have a chance to thoroughly test it. Additionally, there are some styling issues that arise with the use of the `.tippy-content` class.

Regarding the placement optimal solution, I believe that setting `placement: "auto-end"` would be a perfect fit both for desktop and mobile.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/53193850/223944222-59672547-a658-4d9c-9600-56192035d039.png">

Reply:
> How did you pick 335px for this?

I also think that we shouldn't limit the maximum width in this case. Since we're dealing with `<li>` elements that are arranged in a column, their width is fairly predictable and shouldn't need to be excessively wide. Furthermore, setting a `maxWidth` attribute may cause issues if we make changes to the font size or other styling in the future, so I'm planning to remove it.

Reply:
Cool, removing sounds correct, since translations/fonts can change over time.